src/lib/executors/Node.ts

```diff
@@ -875,13 +876,46 @@ export default class Node extends Executor<NodeEvents, Config, NodePlugins> {
         }
 
         testTask
-          .then(() => {
+          .then(async () => {
             if (!this._sessionSuites) {
               return;
             }
-            return this._loadFunctionalSuites().then(
-              () => (testTask = this._runRemoteTests())
-            );
+
+            const allSessions = this._sessionSuites;
+            let suites = allSessions;
```

Comment:
Could we call this `sessions`, and have `failedSessions`? The naming scheme is a bit inconsistent between here and `_runRemoteTests`.